### PR TITLE
Add support for versioner service

### LINKED ISSUE #38: Support Versioner Subserver
Feature request to add support for the [Versioner Service](https://lightning.engineering/api-docs/category/versioner-service).

Not urgent - I'll get around to this myself eventually! Opening to track the dependency elsewhere.

### PR DESCRIPTION
**Pull request by nassersaazi**
This adds support for the [versioner service](https://lightning.engineering/api-docs/category/versioner-service) . This will be useful for the [lndk project](https://github.com/carlaKC/lndk). Closes #38

I used https://github.com/Kixunil/tonic_lnd/pull/33 as a reference, since it makes much the same additions but for the peersrpc service.

Thanks much to anyone able to review

### CODE REVIEW

File: examples/getversion.rs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    let cert_file = args.next().expect("missing arguments: cert file, macaroon file");
+    let macaroon_file = args.next().expect("missing argument: macaroon file");
+    let address = address.into_string().expect("address is not UTF-8");
+
+    // Connecting to LND requires only address, cert file, and macaroon file
+    let mut client = tonic_lnd::connect(address, cert_file, macaroon_file)
+        .await
+        .expect("failed to connect");
+
+    let _version = client
```
Comment (carlaKC) [inline examples/getversion.rs:15]: Don't need the `_` here if we're using `version` below?

File: src/lib.rs [hunk 2]
```diff
@@ -156,6 +166,9 @@ pub mod signrpc {
     tonic::include_proto!("signrpc");
 }
 
+pub mod verrpc {
+    tonic::include_proto!("verrpc");
+}
```
Comment (carlaKC) [inline src/lib.rs:171]: nit: newline between this and `peersrpc`

File: vendor/verrpc/verrpc.proto [hunk 3]
```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
```
Comment (carlaKC) [inline vendor/verrpc/verrpc.proto:1]: Personal preference nitpick: move addition to vendor into a separate commit? Nice to have these changes separated out a little.

**Review by carlaKC — APPROVED** [review]
tACK, tested the example against my local LND. Just some style nits!

**Comment by nassersaazi** [comment]
thanks a lot for the review @carlaKC . i have updated the PR with the suggested changes

**Review by carlaKC — APPROVED** [review]
LGTM!

**Review by Kixunil — APPROVED** [review]
Oh, sorry, I didn't notice this, was quite busy lately. Looks good.

BTW for the future I would prefer a contained change in a single commit and only split unrelated changes. Since the proto file is useless without code changes it makes more sense to put it into single commit. Not a big deal though.
